tests/test_build.py

```diff
@@ -69,3 +69,25 @@ def test_build_module_no_docstring():
             with pytest.raises(common.NoDocstringError) as exc_info:
                 build.main(pyproject)
             assert 'no_docstring.py' in str(exc_info.value)
+
+def test_build_package_with_unicode(copy_sample):
+    list_files_template = """\
+#!{python}
+import sys
+from os.path import join
+if '--deleted' not in sys.argv:
+    files = [
+        'pyproject.toml', 
+        'packageunicode/__init__.py', 
+        u'packageunicode/No\\u00ebl.jpg',
```

Comment:
```suggestion
        'packageunicode/No\\u00ebl.jpg',
```

We don't need to worry about Python 2 here. :-)
We're only

Reply:
I’m trying to figure out why the appveyor ci is failing.  The tests all run successfully locally with tox.